### PR TITLE
Sequence introduced

### PR DESCRIPTION
**Pull request by Riglanto**
- Sequence
- Wagmi upgraded
image

### CODE REVIEW

File: components/Actions.tsx [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+    }
+  }, [data]);
 
   return (
     <form
       className="flex flex-col"
       onSubmit={async (e) => {
         e.preventDefault();
-        await send();
+        transfer!();
```
Comment (Syncretik) [inline components/Actions.tsx:37]: that's cool :D

File: components/Header.tsx [hunk 2]
```diff
@@ -54,12 +54,28 @@ export default function Header({ address, isConnected }: Props) {
           </button>
         </>
       ) : (
-        <button
-          className="primary-button"
-          onClick={() => connect({ connector: connectors[0] })}
-        >
-          Log in
-        </button>
+        <>
```
Comment (Syncretik) [inline components/Header.tsx:57]: do we need a selection here, as the dropdown refactor prompts directly
should we do `Login` default --> sequence without options for smoother UX @gglucass

File: lib/utils.ts [hunk 3]
```diff
@@ -1,55 +1,21 @@
...
-    const userToken = await getUserToken();
-    const appPubKey = !isExternalWallet() ? await getPubKey() : null;
-
     apiInstance = axios.create({
       baseURL: "/api/",
       headers: {
         "Content-Type": "application/json",
-        Authorization: `Bearer ${userToken}`,
-        "glo-app-pub-key": appPubKey,
+        // Authorization: `Bearer ${userToken}`,
```
Comment (Syncretik) [inline lib/utils.ts:17]: remove?

File: lib/utils.ts [hunk 4]
```diff
@@ -1,55 +1,21 @@
...
-
-    return app_pub_key;
-  } catch {
-    return null;
-  }
-};
-
-const isExternalWallet = () =>
-  WALLET_ADAPTERS.OPENLOGIN !== web3AuthInstance.connectedAdapterName;
+// const isExternalWallet = () =>
```
Comment (Syncretik) [inline lib/utils.ts:8]: remove?

File: lib/auth.ts [hunk 5]
```diff
@@ -6,27 +6,7 @@ export const isAuthenticated = async (req: NextRequest) => {
...
-    // https://web3auth.io/docs/pnp/features/server-side-verification/external-wallets
-    const wallet = await decode(idToken, "https://authjs.web3auth.io/jwks");
-
-    if (wallet.address.toLowerCase() === address.toLowerCase()) {
-      return true;
-    }
-  }
-
-  return false;
+  return true;
```
Comment (Syncretik) [inline lib/auth.ts:9]: ehehe

File: components/Header.tsx (hunk 2 above)
Comment (gglucass) [reply components/Header.tsx:57]: I believe this was done like this so we have _something_ in place before @Riglanto went on a mini 2-day holiday.

The high level design should become like here: https://www.figma.com/file/396p2PzjnIXSbHmuBuj7oK/Website?type=design&node-id=3531%3A6970&t=Wug69Uvs9EqQOG9u-1

**Pull request merged**